tests/migrations/test_autodetector.py

```diff
@@ -1495,6 +1499,20 @@ def test_remove_field_and_foo_together(self):
         self.assertOperationAttributes(changes, "otherapp", 0, 1, name="book", unique_together={("author", "title")})
         self.assertOperationAttributes(changes, "otherapp", 0, 2, name="book", index_together={("author", "title")})
 
+    def test_remove_field_that_was_indexed(self):
+        """
+        Removed fields will be removed after updating index/unique_together.
+        """
+        changes = self.get_changes(
+            [self.author_empty, self.book_foo_together], [self.author_empty, self.book_foo_together_5]
+        )
+        # Right number/type of migrations?
+        self.assertNumberMigrations(changes, "otherapp", 1)
+        self.assertOperationTypes(changes, "otherapp", 0, ["AlterUniqueTogether", "AlterIndexTogether", "RemoveField"])
+        self.assertOperationAttributes(changes, "otherapp", 0, 0, name="book", unique_together=set())
+        self.assertOperationAttributes(changes, "otherapp", 0, 1, name="book", index_together=set())
+        self.assertOperationAttributes(changes, "otherapp", 0, 2, model_name="book", name="title")
+
```

Comment:
By the way, I just checked out your branch locally and you might want to adjust this test as it passes against master without your `django.db.migrations.optimizer` changes.

Reply:
Ah, thanks for pointing that out. I was originally piggybacking on a separate test I'll go back and correct it tomorrow.